examples/xgboost_simple.py

```diff
@@ -24,7 +24,6 @@
 import sklearn.metrics
 from sklearn.model_selection import train_test_split
 import xgboost as xgb
-
 import optuna
```

Comment:
This blank line is to separate the third-party library and our library (i.e., optuna), so please revert the change.

Reply:
Please address this comment to pass the `black` checker.